src/index.html

```diff
@@ -8,7 +8,12 @@
 <body>
     <div class="wrap">
       <h1>To-do list</h1>
-      <div id="todo-list"></div>
+      <div id="todo-list">
+        
+      </div>
+      <div class="back">
+        <button id="clear-completed" class="clear-btn">Clear all completed</button>
+    </div>
     </div>
     <script src="./index.js"></script>
```

Comment:
- [ ] In order to use JavaScript Modules, we have to specify the type in the script tag like `<script type="module">` and then we store our module files in the `Modules` directory. So we will have the `index.js` and `status.js` files in the module directory